### DISCUSSION TITLE
Polymorphic One to One Relationships

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11452] [Archive]
I'm amazed at the ease of setting up relationships via the App. Especially, polymorphic relations are such a breeze to work with. via the M2A. 

Is there any way to enforce composite unique indexes in the junction collection (or in general) to ensure that I can create polymorphic one-to-one relations?

**Comment by rijkvanzanten** [comment]
There's currently no support for composite/compound unique/primary keys in Directus yet
